decidim-bulletin-board-client-ruby/LICENSE.txt

```diff
@@ -0,0 +1,21 @@
+The MIT License (MIT)
```

Comment:
is @decidim/product ok with using the MIT license instead of Affero GPL 3?

Reply:
Yes, it should be AGPL3 by default. Also, it's missing at `decidim-bulletin-board-client-js/`